site/content/en/docs/contrib/addons.en.md

```diff
@@ -89,7 +89,7 @@ NOTE: `minikube addons open` currently only works for the `kube-system` namespac
 Rebuild the minikube binary and apply the addon with extra logging enabled:
 
 ```shell
-make && ./out/minikube addons enable <addon name> --alsologtostderr
+make && make test && make lint && ./out/minikube addons enable <addon name> --alsologtostderr
```

Comment:
make test I believe already includes lint

Reply:
true, I will update the PR leaving only `make test`